src/Bridges/Constraint/geomean_to_relentr.jl

```diff
@@ -0,0 +1,92 @@
+"""
+    GeoMeantoRelEntrBridge{T}
+
+The `geometric mean cone` is representable with a relative entropy constraint and a nonnegative auxiliary variable,
+since ``u \\le \\prod_{i=1}^n w_i^{1/n}`` if and only if there exists a ``y \\ge 0`` such that
+``0 \\ge \\sum_{i=1}^n (u + y) \\log (\\frac{u + y}{w_i})``, or equivalently
+``(0, w, (u + y) e) \\in RelativeEntropyCone(1 + 2n)``, where ``e`` is a vector of ones.
+"""
+struct GeoMeantoRelEntrBridge{T, F, G, H} <: AbstractBridge
+    y::MOI.VariableIndex
+    ge_index::CI{F, MOI.GreaterThan{T}} # for y >= 0
+    relentr_index::CI{G, MOI.RelativeEntropyCone}
+end
+function bridge_constraint(::Type{GeoMeantoRelEntrBridge{T, F, G, H}}, model::MOI.ModelLike, f::H, s::MOI.GeometricMeanCone) where {T, F, G, H}
+    f_scalars = MOIU.eachscalar(f)
+    y = MOI.add_variable(model)
```

Comment:
I would do `MOI.add_constrained_variables(model, MOI.Nonnegatives(1))`. Otherwise, you loose problem structure. Here for instance, a solver not supporting free variables will need to bridge `y` with the FreeBridge.